qa/runtests.sh

```diff
@@ -16,22 +16,21 @@ check_python_version() {
 
 echo "Create Virtualenv for Python deps ..."
 function prepare_venv() {
-        # we want tests to run on python3.6
-        printf 'checking alias `python3.6` ... ' >&2
-        PYTHON=$(which python3.6 2> /dev/null)
-        if [ "$?" -ne "0" ]; then
-                printf "%sNOT FOUND%s\n" "${YELLOW}" "${NORMAL}" >&2
-
-                printf 'checking alias `python3` ... ' >&2
-                PYTHON=$(which python3 2> /dev/null)
+        VIRTUALENV=$(which virtualenv)
```

Comment:
Do we want to use a separate virtualenv command, I do not use it. I use the python -m command.

Reply:
This was a standard that pavel had set for all the runtest files. i think he missed out on this one.